frontend/src/metabase/home/components/EmbedHomepage/FeedbackModal.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from "react";
+import { t } from "ttag";
+
+import {
+  Button,
+  Group,
+  Modal,
+  rem,
+  Stack,
+  Text,
+  Textarea,
+  TextInput,
+  Title,
+} from "metabase/ui";
+
+type FeedbackModalProps = {
+  opened: boolean;
+  onClose: () => void;
+  onSubmit: (feedback: { feedback: string; email: string }) => void;
+};
+
+export const FeedbackModal = ({
+  opened,
+  onClose,
+  onSubmit,
+}: FeedbackModalProps) => {
+  const [feedback, setFeedback] = useState("");
+  const [email, setEmail] = useState("");
+  const handleSubmit = () => onSubmit({ feedback, email });
```

Comment:
This isn't the end of the world, but what do you think about adding some very simple input normalization? Now we even send empty strings

![image](https://github.com/metabase/metabase/assets/1937582/da028de5-5c20-4b30-b5a0-e86d87745e0d)


If we do some kind of normalization here, we could skip doing `email || undefined` in the parent component too, but it's also fine to do the normalization in the parent component, but I feel like putting the normalization logic closer to the input components would be better since the input components should know what value they will send back.

Reply:
The be expects either nothing (undefined) or a non empty string, so updated to trim strings and don't send them if empty